multiapps-controller-process/src/main/java/org/cloudfoundry/multiapps/controller/process/jobs/CleanUpJob.java

```diff
@@ -31,8 +30,11 @@
     List<Cleaner> cleaners;
     private final SafeExecutor safeExecutor = new SafeExecutor(CleanUpJob::log);
 
-    @Override
-    public void execute(JobExecutionContext context) {
+    @Scheduled(cron = "#{@cleanUpJob.getCronExpression()}")
```

Comment:
Why not `@Scheduled(cron = "#{@applicationConfiguration.getCronExpressionForOldData()}")`